lib/rspec-stripe/factories/token.rb

```diff
@@ -0,0 +1,48 @@
+module RSpecStripe::Factory
+  class Token
+    attr_accessor :id
+
+    def initialize(id)
+      @id = id || :visa
+    end
+
+    def get
+      @get ||= begin
+        Stripe::Token.create(
+          card: {
+            exp_month: "01",
+            exp_year: "2025",
+            cvc: "111"
+          }.merge(recipes[id])
```

Comment:
Might want to `.to_sym` here to prevent string arguments from crashing. I'll make that change on merge